packages/example-app-base/src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -38,7 +38,7 @@ style['hljs-comment'] = style['hljs-quote'] = {
 };
 
 style['hljs-built_in'] = style['builtin-name'] = {
-  color: FluentTheme.palette.themeDarker,
+  color: FluentTheme?.palette.themeDarker,
```

Comment:
Oh weird, this is not where I thought the error came from--this part is a bit older (1 month) and it really seems like the default FluentTheme ought to be defined...